runtime/src/main/java/org/corfudb/runtime/view/LayoutManagementView.java

```diff
@@ -110,6 +110,36 @@ public void bootstrapNewNode(String endpoint)
         log.info("bootstrapNewNode: New node {} bootstrapped.", endpoint);
     }
 
+    /**
+     * Removes a node from the layout
+     * @param currentLayout the layout to remove the node from
+     * @param endpoint the node to remove
+     */
+    public void removeNode(Layout currentLayout, String endpoint) throws QuorumUnreachableException,
+            CloneNotSupportedException,
+            LayoutModificationException, OutrankedException {
+
+        currentLayout.setRuntime(runtime);
+        sealEpoch(currentLayout);
```

Comment:
move seal after the layout creation.

Reply:
Yes i think you can. We should not seal if we are going to throw an exception.

Reply:
Done.